src/models/activityReportObjectiveCitation.js

```diff
@@ -0,0 +1,39 @@
+const { Model } = require('sequelize');
+
+/**
+   * Junction table between ARO and Citations.
+   * @param {} sequelize
+   * @param {*} DataTypes
+   */
+export default (sequelize, DataTypes) => {
+  class ActivityReportObjectiveCitation extends Model {
+    static associate(models) {
+      ActivityReportObjectiveCitation.belongsTo(models.ActivityReportObjective, {
+        foreignKey: 'activityReportObjectiveId',
+        onDelete: 'cascade',
+        as: 'activityReportObjective',
+      });
+      ActivityReportObjectiveCitation.belongsTo(models.MonitoringStandard, { foreignKey: 'citationId', onDelete: 'cascade', as: 'citation' });
+    }
+  }
+  ActivityReportObjectiveCitation.init({
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: DataTypes.INTEGER,
+    },
+    activityReportObjectiveId: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    citationId: {
+      type: DataTypes.STRING,
+      allowNull: false,
```

Comment:
This is insufficient linkage as this links through the finding, the fining is a live table that will change statuses over time. This will result in inability to properly display the state of the finding for the citation at the time of the report. just a single link is not going to work. more data needs to be cached.